packages/ui-components/src/components/AppShell/AppShell.test.js

```diff
@@ -107,6 +107,19 @@ describe("AppShell", () => {
     expect(screen.queryByText("My Page Footer")).not.toBeInTheDocument()
   })
 
+  test("renders a TopNavigation as passed in embedded mode", async () => {
```

Comment:
Just want to point out async rule that I found it out difficult on a future ts migration.

Reply:
Good point. Unfortunately I think it's everywhere due to copy/paste. But at least we can remove the unecessary ones now every time we touch the tests.